vgc/cameraFilter.py

```diff
@@ -1,82 +1,107 @@
 """Filters a video stream.
-
 """
 import threading
 import cv2
 
 from .filter import Filter
+from .io import outputAdapter
+from . import config
 
 
-class cameraFilter(Filter):
+class CameraFilter(Filter):
     """Filters a video stream
-            
-        This filter rotates and cropps a video stream according to a pitch, a jaw and a zoom variable . 
-        The pitch, jaw and zoom variables are updated asynchronously through the update funciton.
-    
+
+        This filter rotates and cropps a video stream according to
+        a pitch, a jaw and a zoom variable. The pistch, jaw and zoom
+        variables are updated asynchronously through the update funciton.
```

Comment:
This docstring should list functions and variables that belong to the class, add that. See "Dokumentering av kod" for more info. Would be nice with further explanation to the variables.